tools/run_tests/run_performance_tests.py

```diff
@@ -536,7 +536,7 @@ def main():
                              netperf=args.netperf,
                              netperf_hosts=args.remote_worker_host,
                              server_cpu_load=args.server_cpu_load)
-
+  print(scenarios)
```

Comment:
Is this intended?

Reply:
Oh. I was using this to print the benchmark scenario from other languages to see how many channel/RPC per channel they are using.
I'll delete it.